app/notification/model/multi_input_data.py

```diff
@@ -0,0 +1,119 @@
+import uuid
+
+from app.notification.model.notification_utils import convert_bool_value
+
+
+class MultiInput:
+    indent = " " * 5
+
+    @classmethod
+    def format_values(cls, value, index):
+        """
+        Format the given value based on the specified conditions.
+        Args:
+            value: The value to be formatted.
+            index: The index of the value in the processing sequence.
+        Returns:
+            str: The formatted string representation of the value.
+        """
+        return f"{cls.indent}. {value}" if index != 1 else f". {value}"
+
+    @classmethod
+    def format_keys_and_values(cls, key, value, index):
+
+        """
+        Format the given key-value pair based on specified conditions.
+        Args:
+            key: The key of the key-value pair.
+            value: The value associated with the key.
+            index: The index of the key-value pair in the processing sequence.
+        Returns:
+            str: The formatted string representation of the key-value pair.
+        """
+
+        def formatted_values(value):
+            return (
+                ", ".join(
+                    map(
+                        str,
+                        convert_bool_value([value])
+                        if len(value) > 1
+                        else convert_bool_value(value),
+                    )
+                )
+                if isinstance(value, list)
+                else convert_bool_value(value)
+            )
```

Comment:
Not quite sure what is going on here

If the length of the value is > 1 (so we assume its a list), we nest it in another list?